style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #3c3a39;
+  height: auto;
+  justify-content: space-between;
+  flex: 1;
+  padding-bottom: 16px;
+}
+
+.header a {
+  color: #fff;
+  font-family: "Roboto", sans-serif;
+  margin-left: 24px;
+  text-decoration: none;
+}
+
+.header img {
+  height: 24px;
+  width: 24px;
+  margin-right: 32px;
+  margin-bottom: 1px;
+}
+
+.mainContainer {
+  display: flex;
+  background-color: #1c1a19;
+  height: 590px;
+  top: 92px;
+  background-image: url("img/Illustration.png");
+  background-repeat: no-repeat;
```

Comment:
Kindly add the images folder because all the images do not appear here too.

Reply:
All good now 👍🏽